drivers/ipm/Kconfig.imx

```diff
@@ -64,3 +64,10 @@ config IPM_IMX_MAX_ID_VAL
 	default 1 if IPM_IMX_MAX_DATA_SIZE_8
 	default 0 if IPM_IMX_MAX_DATA_SIZE_16
 	depends on IPM_IMX || IPM_IMX_REV2
+
+config IPM_IMX_FW_READY_REPLY
+	bool "FW_READY reply after boot"
+	default n
+	depends on IPM_IMX || IPM_IMX_REV2
+	help
+	  Send FW_READY reply after boot
```

Comment:
after boot? Then why this is in the IPM driver?

Reply:
I'll update the message here - after messaging unit initialization, because I need the MU to be up in order to trigger the interrupt.